docs/asciidoc/modules/ROOT/pages/overview/apoc.load/apoc.load.directory.adoc

```diff
@@ -20,9 +20,10 @@ apoc.load.directory(pattern = * :: STRING?, urlDir =  :: STRING?, config = {} ::
 == Input parameters
 [.procedures, opts=header]
 |===
-| Name | Type | Default 
-|pattern|STRING?|*
-|urlDir|STRING?|
+| Name | Type | Description
```

Comment:
This and the next four files has a comment on top that

////
This file is generated by DocsTest, so don't change it!
////

so I'm not sure which parts can be updated manually and not - again perhaps @vga91 knows and can explain to us

Reply:
As mentioned above, this is automatically generated by the `DocsTest.java`, precisely by `DocumentGenerator.writeInputParametersHeader()` method, so I think the choices are:
- put this part in the generic page: `docs/asciidoc/modules/ROOT/pages/background-operations/apoc-load-directory-async.adoc`
- put in pages with `partial/usage/name.procedure`, not generated
- generate `DocsTest` and roll back these changes